measurements/pages/docs.py

```diff
@@ -8,6 +8,4 @@
 
 @api_docs_blueprint.route('/', methods=["GET"])
 def api_docs():
-    with open(os.path.join(BASE_DIR, 'markdown', 'api_docs.md')) as in_file:
-        docs_text = in_file.read()
-    return render_template('api.html', docs_text=docs_text)
+    return render_template('api.html')
```

Comment:
Does this mean `api_docs.md` can also be removed?

Reply:
Yes `api_docs` can be remove. I have not done it yet, because I want to first ensure I haven't missed anything.

Reply:
okay, I assume this can also be done in another PR